xlml/utils/metric.py

```diff
@@ -545,6 +545,48 @@ def get_gke_job_status(benchmark_id: str) -> bigquery.JobStatus:
   return bigquery.JobStatus.FAILED
 
 
+def get_gke_job_status(
```

Comment:
Have you tested in both cases?

Reply:
Yes, tested for both Will's GKE task and torchbench with postprocess.